subprojects/docs/src/docs/userguide/task_configuration_avoidance.adoc

```diff
@@ -152,12 +155,30 @@ tasks.register("verificationTask") { verificationTask ->
     check.get().dependsOn verificationTask
 }
 ----
+=====
+[.multi-language-sample]
+=====
+[source,kotlin]
+----
+val check = tasks.register("check")
```

Comment:
:x: can be simplified to  `val check by tasks.registering` to avoid repeating the task name